lua/resession/types.lua

```diff
@@ -23,3 +22,7 @@
 ---@field is_win_supported nil|fun(winid: integer, bufnr: integer): boolean
 ---@field save_win nil|fun(winid: integer): any
 ---@field load_win nil|fun(winid: integer, data: any): nil|integer
+
+---@class resession.SessionInfo
+---@field name nil|string Name of the current session
+---@field directory nil|string Name of the directory that the current session is saved in
```

Comment:
`name` should be non-nilable, and I believe `directory` should be as well

Reply:
I set the directory in `session_configs` to the default value if it wasn't provided.